docs/sources/upgrading/_index.md

```diff
@@ -108,6 +108,12 @@ The response body has the following schema:
 * `query_ingesters_within` under the Queier config now defaults to `3h`, it was previously set to 0, meaning always query ingesters.
 * `max_concurrent` under the Querier config now defaults to `10` instead of `20`, since is should be the same as the Frontend Worker's `parallelism` setting (which is `10`).
 
+#### Changed logic of handling RPC error with code "Cancelled"
```

Comment:
As cyril mentioned, let's remove this. We're just fixing a bug introduced that's not in a release.

Reply:
done